src/components/GithubLink/GithubLink.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Icon } from '../Icon/Icon';
+import { Tooltip } from '../Tooltip/Tooltip';
+import { CSSTransition } from 'react-transition-group';
+
+import './GithubLink.scss';
+
+const GithubLink = ({ name, color, size, className }) => {
```

Comment:
I think since `GithubLink` is a very specific component that shouldn't be customized, we don't need to accept props